Refresh.GameServer/Services/GuidCheckerService.cs

```diff
@@ -1,17 +1,21 @@
 using Bunkum.Core.Services;
 using NotEnoughLogs;
 using Refresh.GameServer.Authentication;
+using Refresh.GameServer.Configuration;
 using Refresh.GameServer.Resources;
 
 namespace Refresh.GameServer.Services;
 
 public class GuidCheckerService : EndpointService
 {
+    
+    private readonly GameServerConfig _config;
     private readonly HashSet<long> _validMainlineTextureGuids= [];
     private readonly HashSet<long> _validVitaTextureGuids = [];
 
-    internal GuidCheckerService(Logger logger) : base(logger)
+    internal GuidCheckerService( GameServerConfig config, Logger logger) : base(logger)
```

Comment:
Small formatting fix
```suggestion
    internal GuidCheckerService(GameServerConfig config, Logger logger) : base(logger)
```